<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferPage</class>
 <widget class="QWidget" name="TransferPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>529</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="balanceLabel">
   <property name="geometry">
    <rect>
     <x>692</x>
     <y>25</y>
     <width>271</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="addressLabel2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>65</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(177,177,177);</string>
   </property>
   <property name="text">
    <string>来自地址：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="balanceLabel2">
   <property name="geometry">
    <rect>
     <x>619</x>
     <y>33</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>账户余额</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="addressLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>500</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: rgb(177, 177, 177);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="changeAccountLabel">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>33</y>
     <width>35</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>来自</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>315</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>备注</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="messageLineEdit">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>310</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
  </widget>
  <widget class="QLabel" name="sendtoLabel">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>150</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发往</string>
   </property>
  </widget>
  <widget class="QLabel" name="amountLabel">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>205</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QToolButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>386</y>
     <width>140</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>转账</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amountLineEdit">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>200</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="amountAssetLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>207</y>
     <width>150</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>CDC</string>
   </property>
  </widget>
  <widget class="QLabel" name="feeLabel">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>260</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>手续费</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="feeLineEdit">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>255</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string>0.0001</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="unitLabel2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>262</y>
     <width>150</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>CDC</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel3">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>347</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>余额不足</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>149</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>无效地址</string>
   </property>
  </widget>
  <widget class="QToolButton" name="addContactBtn">
   <property name="geometry">
    <rect>
     <x>476</x>
     <y>147</y>
     <width>64</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="delegateLabel2">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>31</y>
     <width>60</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(40, 40, 40);</string>
   </property>
   <property name="text">
    <string>转账</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel5">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>347</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="accountComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>33</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendtoLineEdit">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>145</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>314</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>长度超过40个字节</string>
   </property>
  </widget>
  <widget class="QLabel" name="changeAccountLabel_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>33</y>
     <width>60</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择资产</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="assetComboBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>33</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="contactLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>149</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 138, 254);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="amountLabel_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>262</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(177,177,177);</string>
   </property>
   <property name="text">
    <string>如需修改手续费请到右上角设置窗口</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>amountLineEdit</tabstop>
  <tabstop>feeLineEdit</tabstop>
  <tabstop>messageLineEdit</tabstop>
  <tabstop>sendBtn</tabstop>
  <tabstop>addContactBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
